packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.6.7+8
 
 * Update documentation of getImage() about Android's disability to preference front/rear camera.
+* Updated documentation with code that does not throw an error when image is not picked.
```

Comment:
Please make it a new version as 0.6.7+8 has been published.

Reply:
Done. I have made it as v0.6.7+10